src/kubo/utils.ts

```diff
@@ -66,6 +66,10 @@ export function buildStartArgs (options: KuboStartOptions): string[] {
     args.push('--offline')
   }
 
+  if (options.pubsub === true) {
+    args.push('---enable-pubsub-experiment')
```

Comment:
3 dashes rather than 2

Reply:
There was a test for this, but, well:

![tests](https://i.makeagif.com/media/9-27-2021/7Q2874.gif)

Fixed in https://github.com/ipfs/js-ipfsd-ctl/commit/bb5f1a1c8f23415a40de34de80224370cf717157 released in https://github.com/ipfs/js-ipfsd-ctl/releases/tag/v14.0.5